src/include/distributed/worker_transaction.h

```diff
@@ -22,9 +22,8 @@
  */
 typedef enum TargetWorkerSet
 {
```

Comment:
minor: Maybe expand the comment above stating that we never add coordinator to the  return list.